valhalla/mjolnir/landmark_builder.h

```diff
@@ -60,7 +60,7 @@ enum class LandmarkType : uint8_t {
   casino = 18,
 };
 
-using Landmark = std::tuple<std::string, LandmarkType, double, double>;
+using Landmark = std::tuple<int64_t, std::string, LandmarkType, double, double>;
```

Comment:
Hm I just realized this: not that we really need it but why not uint64_t for the index?

Reply:
because the primary key has to be a signed 64 bit integer to get a free index on that column in sqlite. why they picked signed i do not know but if we want to use unsigned that means we have to add our own index. anyway this is what i read on the internet. if we were going to do something like that we should forget the primary key and use the osmid directly and add an index manually (this could be useful for debugging at some point?) but i figured at the moment its not needed, it did cross my mind though!